kernel/kernel-api/src/main/java/io/delta/kernel/internal/TransactionImpl.java

```diff
@@ -149,6 +147,21 @@ public TransactionCommitResult commit(Engine engine, CloseableIterable<Row> data
                             "New commit version %d should be greater than the previous commit " +
                                     "attempt version %d.", newCommitAsVersion, commitAsVersion);
                     commitAsVersion = newCommitAsVersion;
+                    Optional<Long> updatedInCommitTimestamp =
```

Comment:
Did we want these ICT changes in a separate PR?

Reply:
It's caused by a stacked PR. That PR has been merged now.